.github/pull_request_template.md

```diff
@@ -1 +1,96 @@
+Please select the template that's relevant for your situation and delete the other templates.
 
+# :label: Bump for version `VERSION_NUMBER`
+
+## What changes does this release include?
+
+The easiest and most reliable way to find the changes that this release includes is to go through the changes between the last version number and master: `https://github.com/NoRedInk/noredink-ui/compare/VERSION_NUMBER...master`.
+
+Include links to the relevant PRs in a list in this PR. Be sure to note any risky changes or changes that you will want to alert QA to when upgrading to the new version of noredink-ui.
+
+Please update the PR's name to include a quick note about every linked PR.
+
+## How has the API changed?
+
+Please paste the output of `elm diff` run on latest master in the code block:
+
+```
+
+```
+
+## Releasing
+
+After this PR merges, and you've pulled down latest master, finish following the [publishing process](https://github.com/NoRedInk/noredink-ui/blob/master/README.md#publishing-a-new-version).
+
+
+---
+
+# :star2: Adding a new component
+
+## About the component
+
+What is the purpose of the new component? When and where will it be used? How will the reviewer know if this component meets the success criteria?
+
+Please link to any relevant context and stories.
+
+
+## :framed_picture: What does this change look like?
+
+- Fill in with screenshots and ping design.
+- [Guidelines for an optimal design ping](https://paper.dropbox.com/doc/Guidelines-for-Sharing-User-Facing-Changes-with-Design--BpL8hpJLMugy6033aT5m0JdaAg-bdKGQtYH9qO9I00hUkA6k)
```

Comment:
```suggestion
- Follow the the [Guidelines for an optimal design ping](https://paper.dropbox.com/doc/Guidelines-for-Sharing-User-Facing-Changes-with-Design--BpL8hpJLMugy6033aT5m0JdaAg-bdKGQtYH9qO9I00hUkA6k) to add helpful screenshots/videos for design.
- Ping design on this PR to get approval on your changes.
```

I don't think it was super clear that design approval is _required_, and that the guidelines are not optional. I also took a look at the guidelines and see that they are actually helpful for understanding what kinds of screenshots/videos to include in the first place, so figured those guidelines should be reviewed before devs take the time to get screenshots/video.